battenberg/cli.py

```diff
@@ -102,5 +103,12 @@ def upgrade(ctx, **kwargs):
     try:
         battenberg = Battenberg(open_repository(ctx.obj['target']))
         battenberg.upgrade(**kwargs)
+    except MergeConflictException:
+        # Just run "git status" in a subprocess so we don't have to re-implement the formatting
+        # logic atop pygit2.
+        completed_process = subprocess.run(['git', 'status'], stdout=subprocess.PIPE,
+                                           stderr=subprocess.STDOUT)
+        print(completed_process.stdout.decode('utf-8'))
```

Comment:
should this use the logger/click instead?

Reply:
Always forget about `click.echo`, fixed in the next commit.